test/scripts/e2e_subs/tealprogs/abi-min_balance.teal

```diff
@@ -0,0 +1,110 @@
+#pragma version 6
+    // TODO - some explanation
```

Comment:
please please please add the explanation ...

Reply:
👍

Reply:
Brief explanation added